content/applications/finance/accounting/others/inventory/avg_price_valuation.rst

```diff
@@ -1,167 +1,477 @@
-| Operation                                     | stock input   | price diff   | Inventory Value   | Qty On Hand   | Avg Cost   |
-+===============================================+===============+==============+===================+===============+============+
-|                                               |               |              | $0                | 0             | $0         |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-| Receive 8 Products at $10                     | ($80)         |              | $80               | 8             | $10        |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-| Receive vendor bill $80                       | $0            |              | $80               | 8             | $10        |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-| Receive 4 Products at $16                     | ($64)         |              | $144              | 12            | $12        |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-| Receive vendor bill $64                       | $0            |              | $144              | 12            | $12        |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-| Deliver 10 Products                           | $0            |              | $24               | 2             | $12        |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-| Return of 1 Product initially bought at $10   | **$10**       | **$2**       | **$12**           | 1             | $12        |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-| Receive vendor refund $10                     | $0            | $2           | $12               | 1             | $12        |
-+-----------------------------------------------+---------------+--------------+-------------------+---------------+------------+
-
-This is because the vendor refund will be made using the original
-purchase price, so to zero out the effect of the return in the stock
-input in last operation, we need to reuse the original price. The price
-difference account located on the product category is used to book the
-difference between the average cost and the original purchase price.
+=========================================
+Calculate average price on returned goods
+=========================================
+
+.. _inventory/avg_cost/definition:
+
+<<<<<<< HEAD
```

Comment:
```suggestion
<<<<<<< HEAD
```
There are some issues with the multiple commits made. No worries, this is part of the learning process. I'll let you see with your GitHub/PyCharm guru how to fix those. You can ping me again for the final green light once everything's fixed :)